src/Kdyby/Translation/DI/TranslationExtension.php

```diff
@@ -220,15 +253,44 @@ protected function loadDumpers()
 	protected function loadLoaders()
 	{
 		$builder = $this->getContainerBuilder();
+		$config = $this->getConfig();
+
+		$loaders = $config['database']['loaders'];
+		if ($loaders === NULL) {
+			$loaders = [];
+		}
+		if (!is_array($loaders)) {
+			$loaders = [$loaders];
+		}
 
 		foreach ($this->loadFromFile(__DIR__ . '/config/loaders.neon') as $format => $class) {
-			$builder->addDefinition($this->prefix('loader.' . $format))
-				->setClass($class)
-				->addTag(self::LOADER_TAG, $format);
+			if (self::getClassReflection($class)->implementsInterface('Kdyby\Translation\Loader\IDatabaseLoader')) {
```

Comment:
tohle by asi bylo rozumny zjednodušit na `in_array('Kdyby\Translation\Loader\IDatabaseLoader', class_implements($class), TRUE)`